lib/proxy/peer/quicserver.go

```diff
@@ -39,10 +52,10 @@ type QUICServerConfig struct {
 
 	// CipherSuites is the set of TLS ciphersuites to be used by the server.
 	//
-	// Note: it won't actually have an effect, since QUIC always uses (the DTLS
-	// equivalent of) TLS 1.3, and TLS 1.3 ciphersuites can't be configured in
-	// crypto/tls, but for consistency's sake this should be passed along from
-	// the agent configuration.
+	// Note: it won't actually have an effect, since QUIC always uses TLS 1.3,
+	// and TLS 1.3 ciphersuites can't be configured in crypto/tls, but for
+	// consistency's sake this should be passed along from the agent
+	// configuration.
 	CipherSuites []uint16
```

Comment:
If this has no effect then what is the point in having it?

Reply:
I thought it was the easiest way to prevent "Why aren't you using `utils.TLSConfig`?" questions.

Reply:
Kill the field and move the comment to type itself, or next to the appropriate initialization?